eth/catalyst/api.go

```diff
@@ -291,6 +291,10 @@ func (api *ConsensusAPI) forkchoiceUpdated(update engine.ForkchoiceStateV1, payl
 		}
 		return engine.STATUS_SYNCING, nil
 	}
+	// Verify Aggregate Signature if there are any BLS transactions in a block
+	if err := engine.VerifyAggregate(block); err != nil {
```

Comment:
I tried following the payload validation flow as described [here](https://hackmd.io/@danielrachi/engine_api#INVALID). 

Which led me to placing this logic here